### ISSUE TITLE
any plan for Network Triggered Service Request?

### ISSUE THREAD

**Issue by congzee** [travelping/upg-vpp#312]
I found that network triggered service request is not available with upg-vpp

For network triggered service request, UPF need to notify SMF about downlink packet being buffered 
in the form of Downlink Data Report IE within PFCP Session Report Request.
**upg-vpp do NOT buffer the packets and notify the CP function but just drop it.**

SMF sent update FAR with apply actions: NOCP and BUFF
but I cannot find source for the apply actions .

Does anyone have plan or idea for implementation? Please?🙏

**Comment by RoadRunnr** [comment]
You are right, that feature is not supported. We do not have plans to work on it.

Of course, contributions are welcome.
